Bio/AlignIO/MafIO.py

```diff
@@ -35,7 +35,12 @@
 import os
 
 from itertools import islice
-from sqlite3 import dbapi2
+
+try:
+    from sqlite3 import dbapi2
+    from sqlite3.dbapi2 import OperationalError, DatabaseError
+except ImportError:
+    dbapi2 = OperationalError = DatabaseError = None
```

Comment:
I don't think we need to define ``OperationalError`` and ``DatabaseError`` here.

Why not just revert the changes from 4e77946e4c3c711e35d4a4ad40f52ae1dbc5e3bb on this file?

Reply:
That works for me too, I was just following the `MissingPythonDependencyError` used elsewhere.